bittensor_cli/cli.py

```diff
@@ -267,6 +268,12 @@ class Options:
         "--dashboard.path",
         help="Path to save the dashboard HTML file. For example: `~/.bittensor/dashboard`.",
     )
+    json_output = typer.Option(
+        False,
+        "--json-output",
+        "--json-out",
```

Comment:
```suggestion
        "--json",
```
I think this should be fine and looks more elegant and is similar to other tools

Reply:
No. Because we also use `--json` to refer to JSON files for key generation.